app/component/ItineraryDetailView.js

```diff
@@ -72,7 +72,7 @@ const ItineraryShape = PropTypes.shape({
 });
 
 /* eslint-disable prettier/prettier */
-class ItineraryTab extends React.Component {
+class ItineraryDetailView extends React.Component {
```

Comment:
What is the difference between view and page? Also, not sure if it should be Details or Detail. The old name sort of implied this is a swipeable tab (maybe by accident), not sure if that is relevant.

Reply:
Component does not know (and should not know) that it is rendered into a 'tab'.  So Tab should be removed.

I renamed the component as ItineraryDetails.